Monika After Story/game/script-greetings.rpy

```diff
@@ -1593,6 +1595,56 @@ label greeting_back_from_school:
              m 1ekbfa "I love you so, so much."
      return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_back_from_game",
+            unlocked=True,
+            category=[store.mas_greetings.TYPE_GAME],
+            random=True
+        ),
+        eventdb=evhand.greeting_database
+    )
+
+label greeting_back_from_game:
+     m 1hua "Oh, welcome back, [player]!"
+     menu:
+         m "Did you enjoy well with your game?"
+         "Yes.":
+             m 1hub "That's good!"
+             m 1eua "Remember to get some break, okay?"
+             m 4esa "Games can be addictive. Don't let games to siphoning your all time."
```

Comment:
"Games can be addictive. Don't let games siphon all your time."

A correction has been provided, however personally, I think this line has Monika needlessly misunderstand and dismiss an very common lifestyle among people. I personally, would find it annoying to be told that every time I got back from another game. You could take an approach similar to monika_otaku where Monika acknowledges that there are people with that lifestyle, while also being respectful of those people and the player is they happen to have that lifestyle themselves.